addons/website_slides_survey/tests/test_course_certification_unlink.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo.tests import tagged
+from odoo.exceptions import ValidationError
+from odoo.addons.survey.tests.common import TestSurveyCommon
```

Comment:
Should be sorted
https://www.odoo.com/documentation/14.0/fr/developer/misc/other/guidelines.html#imports